app/views/search/_view_all_link.html.erb

```diff
@@ -1,10 +1,10 @@
 <% if current_page?(:controller => 'search', :action => 'search_boxed') %>
   <div class="results-more">
-  <% if @results['total'] > 0 %>
+  <% if @results['total'].to_i.present? %>
```

Comment:
I'm unclear on what would happen here if `@results['total']` can't be cast into an integer for some reason, but that seems like a highly unlikely scenario, and Sentry will let us know if it happens anyway.

Reply:
Yeah. I was wary to look too deep as you are correct and this may be problematic... 🤞🏻 

If this leads to more trouble than it solves, we can take more time and fix it right.